hw/ip/aes/lint/aes.waiver

```diff
@@ -3,3 +3,5 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # waiver file for aes
+waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[*\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
+      -comment "regular[*] is assigned in a for loop, regular[*][1|2|5] depends on regular[*][0|1|4]"
```

Comment:
I wasn't sure about the AscentLint syntax. Could you please have a look @msfschaffner and let me know if this works.

Reply:
Thanks, @vogelpi. Can you use the regex below? I had to change the expression in the first `[]` bracket to make this work.
```
waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[[0-9]+\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
      -comment "regular[*] is assigned in a for loop, regular[*][1|2|5] depends on regular[*][0|1|4]"
```